src/baldr/graphreader.cc

```diff
@@ -237,32 +242,42 @@ const GraphTile* GraphReader::GetGraphTile(const GraphId& graphid) {
     // Do we have this tile
     auto t = tile_extract_->tiles.find(base);
     if (t == tile_extract_->tiles.cend()) {
+      // LOG_DEBUG("Memory map cache miss " + GraphTile::FileSuffix(base));
       return nullptr;
     }
 
     // This initializes the tile from mmap
     GraphTile tile(base, t->second.first, t->second.second);
     if (!tile.header()) {
+      // LOG_DEBUG("Memory map cache miss " + GraphTile::FileSuffix(base));
       return nullptr;
     }
+    // LOG_DEBUG("Memory map cache hit " + GraphTile::FileSuffix(base));
```

Comment:
did you intend to leave in these log messages?

Reply:
in practice they were too chatty. i left them because they are useful for debugging this specific thing but i thought they were too much to always have on. trace would probably be more appropriate